lib/tapjoy/autoscaling_bootstrap.rb

```diff
@@ -153,7 +154,9 @@ def confirm_config(keypair:, zones:, security_groups:, instance_type:,
 
       # configure environment
 
-      def configure_environment(filename, env=nil)
+      def configure_environment(opts)
+        filename = opts[:filename]
+        env = opts[:env] if opts.key?(:env)
```

Comment:
The reason being - now there is ONE place where "env" is set from the multitude of incoming options. Instead of someone coming along later, making a change between this line and one of the others, and having that not be the right place to override something, you have the entire fall-through logic of how to correctly get the env value in one single place.